style.css

```diff
@@ -28,6 +36,15 @@ h2 {
   margin: 0;
 }
 
+.header-2 {
+  font-size: 2.5rem;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 3.25rem;
+
+  /* 130% */

```

Comment:
- [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

Reply:
> - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍